Modules/_datetimemodule.c

```diff
@@ -1578,6 +1578,7 @@ wrap_strftime(PyObject *object, PyObject *format, PyObject *timetuple,
     PyObject *result = NULL;            /* guilty until proved innocent */
 
     PyObject *zreplacement = NULL;      /* py string, replacement for %z */
+    PyObject *colonzreplacement = NULL; /* py string, replacement for %:z */
```

Comment:
Does the comment on line 1269 also need updating?

Reply:
I don't see a comment in 1269.

Reply:
ah, guess you meant 1569?